src/fuzzing/streamStateFromFormat/dict

```diff
@@ -0,0 +1,10 @@
+kw1="%"
+kw2="#"
+kw3="0"
+kw4=" "
+kw5="+"
+kw5="."
+kw5="*"
+kw5="l"
+kw5="u"
+kw5="E"
```

Comment:
Same comment as before about single values, not enough of a problem to interrupt here though.